source/spirv_target_env.h

```diff
@@ -46,4 +48,42 @@ std::string spvLogStringForEnv(spv_target_env env);
 //        occur to satisfy this limit.
 std::string spvTargetEnvList(const int pad, const int wrap);
 
+// Reads the target environment from the header comments of disassembly. Returns
+// true if valid name found, false otherwise.
+bool spvReadEnvironmentFromText(std::vector<char>& text, spv_target_env* env);
+
+static constexpr std::pair<const char*, spv_target_env> spvTargetEnvNameMap[] =
```

Comment:
I had to check, but C++17 makes this inline: multiple definitions can be seen (by any includer of this header) and there will only be one copy of the data.

I'd prefer to spell that out more explicitly by writing `inline` here instead of `static`.

Reply:
Right, but in the original it was in a .cpp file.  So there was only ever going to be one definition of it.
Putting it in the header means you have to be more careful about ensuring there aren't repeat definitions.